controller/Dockerfile

```diff
@@ -26,6 +26,7 @@ CMD ls $(dirname ${LOG_FILEPATH}) && ./test-controller \
     --test-volume-mountpoint=${TEST_VOLUME_MOUNTPOINT} \
     --test=${TEST_NAME} \
     --test-image-name=${TEST_IMAGE_NAME} \
+    --chit-spammer-image-name=${CHIT_SPAMMER_IMAGE_NAME} \
```

Comment:
Does this mean we're going to have a separate Byzantine image (and a separate flag here) for every Byzantine test??

Reply:
yep. unless you somehow change the byzantine repo to be a swiss army knife where you can configure the behavior via API calls but thats something I'm considering out of scope for this